docs/README.md

```diff
@@ -173,50 +173,49 @@ $ ARCH=aarch64 scripts/mk.image
-|`LOGDNA_REDACT_REGEX`|Comma separated list of regex patterns used to mask matching sensitive information (such as PII) before sending it in the log line.||
-|`LOGDNA_JOURNALD_PATHS`|Comma separated list of paths (directories or files) of journald paths to monitor||
-|`LOGDNA_LOOKBACK`|The lookback strategy on startup|`none`|
-|`LOGDNA_K8S_STARTUP_LEASE`|Determines whether or not to use K8 leases on startup|`never`|
-|`LOGDNA_USE_K8S_LOG_ENRICHMENT`|Determines whether the agent should query the K8s API to enrich log lines from other pods.|`always`|
-|`LOGDNA_LOG_K8S_EVENTS`|Determines whether the agent should log Kubernetes resource events. This setting only affects tracking and logging Kubernetes resource changes via watches. When disabled, the agent may still query k8s metadata to enrich log lines from other pods depending on the value of `LOGDNA_USE_K8S_LOG_ENRICHMENT` setting value.|`never`|
-|`LOGDNA_LOG_REPORTER_METRICS`|Determines whether or not metrics usage statistics is enabled.|`never`|
-|`LOGDNA_DB_PATH`|The directory in which the agent will store its state database. Note that the agent must have write access to the directory and be a persistent volume.|`/var/lib/logdna`|
-|`LOGDNA_METRICS_PORT`|The port number to expose a Prometheus endpoint target with the [agent internal metrics](INTERNAL_METRICS.md).||
-|`LOGDNA_INGEST_TIMEOUT`|The timeout of the API calls to the ingest API in milliseconds|`10000`|
-|`LOGDNA_INGEST_BUFFER_SIZE`|The size, in bytes, of the ingest data buffer used to batch log data with.|`2097152`|
-|`LOGDNA_RETRY_DIR`|The directory used by the agent to store data temporarily while retrying calls to the ingestion API.|`/tmp/logdna`|
-|`LOGDNA_RETRY_DISK_LIMIT`|The maximum amount of disk space the agent will use to store retry data. The value can be the total number of bytes or a human representation of space using suffixes "KB", "MB", "GB" or "TB", e.g. `10 MB` If left unset, the agent will not limit disk usage. If set to `0`, no retry data will be stored on disk.||
-|`LOGDNA_META_APP`|Overrides/omits `APP` field in log line metadata. [Examples](META.md)||
-|`LOGDNA_META_HOST`|Overrides/omits `HOST` field in log line metadata.||
-|`LOGDNA_META_ENV`|Overrides/omits `EMV` field in log line metadata.||
-|`LOGDNA_META_FILE`|Overrides/omits `FILE` field in log line metadata.||
-|`LOGDNA_META_K8S_FILE`|Overrides/omits `FILE` field in k8s log line metadata. Follows `LOGDNA_META_FILE`.||
-|`LOGDNA_META_JSON`|Overrides/omits `META` filed in log line metadata.||
-|`LOGDNA_META_ANNOTATIONS`|Overrides specific kay-value-pairs inside `ANNOTATIONS` field in log line metadata.||
-|`LOGDNA_META_LABELS`|Overrides specific kay-value-pairs inside `LABELS` field in log line metadata.||
+configuration YAML file. The default configuration yaml file is located at `/etc/logdna/config.yaml`. Unless noted, when an option says *List of*, this would mean a comma separated list for Environment variable and a YAML array list for YAML.
```

Comment:
v1 format not mentioned? .conf

Reply:
Can you suggest wording here, @dkhokhlov?  Should I mention the windows logdna.conf location?

If I am to list the v1 names...how do I find them?

Reply:
"
For backward compatibility [agent v1 configuration file format](https://github.com/logdna/logdna-agent#configuration-file) is still supported. For example upgrade to Agent v2 on Windows can re-use v1 config file from previous version as-is.
"
@jorgebay  defined v1 config parameter name correspondence [here](https://github.com/logdna/logdna-agent-v2/blame/97237770385f471aed2d5b50059e98ea158feab5/common/config/src/properties.rs#L30).